src/sentry/workflow_engine/endpoints/organization_detector_workflow_index.py

```diff
@@ -0,0 +1,142 @@
+from django.db import IntegrityError
+from drf_spectacular.utils import extend_schema
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import OrganizationEndpoint
+from sentry.api.bases.organization import OrganizationPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_CONFLICT,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.parameters import GlobalParams
+from sentry.workflow_engine.endpoints.serializers import DetectorWorkflowSerializer
+from sentry.workflow_engine.endpoints.validators.detector_workflow import DetectorWorkflowValidator
+from sentry.workflow_engine.models.detector import Detector
+from sentry.workflow_engine.models.detector_workflow import DetectorWorkflow
+from sentry.workflow_engine.models.workflow import Workflow
+
+
+@region_silo_endpoint
+class OrganizationDetectorWorkflowIndexEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+        "DELETE": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ISSUES
+    permission_classes = OrganizationPermission
```

Comment:
does a user need project permission to the detectors in order to connect them?

Reply:
i think having the `project_detector` URL pattern makes sense for POST since that's connecting an single detector to a single workflow, but if GET and DELETE are meant for bulk actions then this wouldn't work since workflows can be connected to detectors from multiple projects 🤔  this is tricky

Reply:
Update: decided that creation/deletion of DetectorWorkflow is determined by detector permissions